README.md

```diff
@@ -84,41 +85,42 @@ drwxr-xr-x  27 adayal  wheel    864 Oct 25  2023 lib
 ```
 
 ---
-## Identify Local Caching Issues using Same Location
+## Identify Local Build Caching Issues (Same Location)
 
-Let's run experiment 02 to identify local caching issues by running commands in the same location.
+Let's run experiment 2 to identify local caching issues when the build is run from the same location.
 
-1. Go to the directory where the build validation scripts are installed
+1. Go to the directory where the Develocity Build Validation Scripts are installed
 
-2. Run the script in interactive mode: `./02-validate-local-build-caching-same-location.sh -i`
+2. Run the second script in interactive mode: `./02-validate-local-build-caching-same-location.sh -i`
 
-3. Press the `<enter>` key to start the experiment as the following prompt indicates:
+3. Press the <kbd>Enter</kbd> key to start the experiment as the following prompt indicates:
 
 ```bash
 Press <Enter> to get started with the experiment.
 ```
 
+[//]: # (todo why does the project not use CCUD?)
```

Comment:
@akshayabd just as the `todo` asks.

Reply:
Since the CCUD is optional, I did not include it in this scenario.

Do you feel strongly about including it? How important is it that we encourage users to apply the plugin?

Reply:
We agreed to apply CCUD to the build.